frontend/src/metabase/visualizations/click-actions/drills/UnderlyingRecordsDrill.unit.spec.js

```diff
@@ -187,6 +188,20 @@ describe("UnderlyingRecordsDrill", () => {
       const [action] = actions;
       expect(action.title).toEqual("See these SomeTitles");
     });
+
+    // NOTE: this test is valid only until lib drill returns metric value instead of underlying rows count
+    it("should return correct pluralized title for negative numeric values", () => {
```

Comment:
Can you add `(metabase#32108)` to the test name so we know this is a test that verifies the fix for that issue?